application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadOverviewUpdater.java

```diff
@@ -39,7 +41,7 @@
 public final class HelpThreadOverviewUpdater extends MessageReceiverAdapter implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadOverviewUpdater.class);
 
-    private static final String STATUS_TITLE = "## __**Active questions**__ ##";
+    private static final @NonNls String STATUS_TITLE = "## Active questions ##";
```

Comment:
The presence of this annotation feels super random and confusing. Mostly because it is the only time its used yet. To prevent people from confusion, Id suggest to remove it (or to add it everywhere it applies instead).

That said, it doesnt even seem correct. This string **is** user facing. It technically would need localization. The annotation says the opposite.

Reply:
I'll remove the annotation.

Initially I was trying or nonce's work, those aren't localised, but it didn't work and forgot to remove afterwards.